cns/restserver/ipam.go

```diff
@@ -83,6 +101,7 @@ func (service *HTTPRestService) requestIPConfigHandlerHelper(ctx context.Context
 		}
 	}
 
+	podIPInfo = append(podIPInfo, podBackendInfo)
```

Comment:
should this go inside the if in line 51? Do we want to have an empty podBackendInfo in the slice in case no backendinterface info is passed in the request?

Reply:
fixed